packages/logger/README.md

```diff
@@ -45,6 +45,16 @@ npm install @sapphire/plugin-logger @sapphire/framework
 import '@sapphire/plugin-logger/register';
 ```
 
+```typescript
+import { container } '@sapphire/framework';
+container.logger.log('log message');
+```
+
+```typescript
+// ping command
```

Comment:
More context is required here for clarity.

Reply:
done.